### ISSUE TITLE
Weird Hand and Hotbar Bug with Volume Boxes

### ISSUE THREAD

**Issue by DrakoAlcarus** [BuildCraft/BuildCraft#4246]
**BuildCraft version:** 7.99.17
**Forge version:** 14.23.4.2747
**Link to crash report or log:** No report, but screenshot: https://imgur.com/a/sAhoF9K
**Singleplayer or multiplayer:** Singleplayer

**Steps to reproduce:** Place land marks and link them or place the item.volume_box.name (which I do not know how to remove...)

**Additional information:** The hand seems to become 4th dementionally large, making my brain collapse as the hand and the item in it clips through everything. The hotbar also was acting odd, now all it does is show all items in hotbar as if they weren't shadowed.

**Comment by DrakoAlcarus** [comment]
**Note to add:** Some rendered blocks do this as well. The list is: Pump Pipe; All Engines; Distiller

Placing these blocks would make the hand do the thing.

**Comment by AlexIIL** [comment]
Are you using any render affecting mods like optifine? If you are than that would provide a direct way of reproducing.

**Comment by DrakoAlcarus** [comment]
I am using Better FPS and Optifine. Recent computer issues led to use them.

Messing with some options don't change anything. Probably with the mods themselves and how they interact. For Better FPS, I'm using Riven's "Full" Algorithm.

**Comment by DrakoAlcarus** [comment]
Also FoamFix, but not sure if that's important

**Comment by AlexIIL** [comment]
Can you try temporarily disabling both of them? It would be very useful if I know for certain that it is an integration with betterfps or optifine. Although if you can narrow it down to a single option that would be even more useful

**Comment by DrakoAlcarus** [comment]
Yeah, no problem. Bit busy for the time being, but will get to it when I
can.

**Comment by DrakoAlcarus** [comment]
Were you able to reproduce this?

**Comment by AlexIIL** [comment]
Sorry, I haven't got around to testing this yet :(
